### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by AndriyTykhan**
Document logical id generation for path variables in urls

Closes: #3343

### CODE REVIEW

**Comment by AndriyTykhan** [comment]
@pmuens  could you review it?

File: docs/providers/aws/guide/resources.md [hunk 1]
```diff
@@ -67,6 +67,8 @@ All resource names that are deployed by Serverless have to follow this naming sc
 
 We're also using the term `normalizedName` or similar terms in this guide. This means dropping any characters that aren't allowed in resources names, e.g. special characters.
 
+If you have path variables in your url, they get normalized too, and there is a `Var` added implicity. So `normalizedPath` for `POST /users/{user_id}` will be normalized to `UsersUseridVarTestPost`
```
Comment (medikoo) [inline docs/providers/aws/guide/resources.md:70]: *implicitly

**Review by medikoo — CHANGES_REQUESTED** [review]
Thank you @AndriyTykhan ! That looks good, I've pointed just one small typo.

Concerning Hackoctoberfest we do not officially participate, but we welcome and value quality contributions at any time.

File: docs/providers/aws/guide/resources.md (hunk 1 above)
Comment (AndriyTykhan) [reply docs/providers/aws/guide/resources.md]: Thanks, I've fixed it

**Review by medikoo — APPROVED** [review]
Thank you @AndriyTykhan !

**Pull request merged**